<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoFileChooseDialog</class>
 <widget class="QWidget" name="VideoFileChooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CHOOSE_VIDEO_FILE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Resources/ltorrent.qrc">
    <normaloff>:/icons/app.ico</normaloff>:/icons/app.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="tbMenu">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../Resources/ltorrent.qrc">:/icons/app.ico</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LTitle">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>CHOOSE_VIDEO_FILE</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_centralWidget" native="true">
     <layout class="QGridLayout" name="mainLayout" rowstretch="2,0">
      <item row="1" column="0" colspan="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/ltorrent.qrc"/>
  <include location="../../../Resources/ltorrent.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>SendReport()</slot>
 </slots>
</ui>
